src/AuthEncoding/AuthEncoding.py

```diff
@@ -162,6 +162,42 @@ def validate(self, reference, attempt):
 registerScheme(u'SHA256', SHA256DigestScheme())
 
 
+# Bcrypt support may not have been requested at installation time
+# - installed via the 'bcrypt' extra
+try:
+    import bcrypt
+except ImportError:
+    bcrypt = None
+
+
+if bcrypt is not None:
```

Comment:
As in https://github.com/zopefoundation/zope.password/pull/5#pullrequestreview-2450238, please update to define the class unconditionally, but raise in `__init__()` if `bcrypt` is None.  That would require a conditional test of that constructor, too.

I don't see any added tests in this PR:  do the parameterized tests actually cover everything for the new handler?